fitbenchmarking/fitbenchmarking
# Adding docs for NIST prob files

#### Description of Work

Fixes #456 

Adds NIST problem definition documentation.


#### Testing Instructions

1. Check the docs are adequate 

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
